pages/dkp/kommander/2.2/legal/version-policy/index.md

```diff
@@ -5,6 +5,7 @@ title: Version Support Policy
 menuWeight: 5
 excerpt: Supported version policy for Kommander
 beta: false
+techPreview: true
```

Comment:
Is this just to test the new badge?   (Which looks great, BTW!). Because otherwise, I don't think we want this page to be considered a 'preview'. :-)

Reply:
Yep! Casie was trying to test it out. We can remove that line when this is ready to go. :)